docs/sources/alerting/fundamentals/notification-policies/index.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Notification Policies
+description: Introduction to Notification Policies and how they work
+weight: 409
+keywords:
+  - grafana
+  - alerting
+  - notification policies
+---
+
+# How Notification Policies work
+
+If you’ve ever used Prometheus’ Alertmanager you might be familiar with its routes and routing tree. It’s a very flexible way to route alerts to various different receivers and, with its label matchers, is a powerful abstraction that allows modifications to alert notification delivery without having to update every individual alert rule.
+
+In Grafana Alerting we refer to these routes as **Notification Policies**.
+
+## A policy tree
+
+Notification policies are _not_ a list but instead a [tree structure](https://en.wikipedia.org/wiki/Tree_structure). This means that each policy could have child policies and so on. The root of the notification policies tree is called the **Default notification policy**.
+
+Each policy consists of a set of label matchers (0 or more) that specify which labels they are or aren't interested in handling.
+
+If you haven't already, check out our fundamental guide on [how label matching works]({{< relref "../annotation-label/labels-and-label-matchers.md" >}}).
+
+{{% admonition type="note" %}}
+When no label matchers are configured for a policy, it will match _all_ alert instances. Take extra care with these policies are they might prevent other policies from being evaluated unless **`continue matching siblings`** is enabled on the notification policy.
```

Comment:
```suggestion
If you haven't configured any label matchers for your notification policy, your notification policy will match _all_ alert instances. This may prevent child policies from being evaluated unless you have enabled **`continue matching siblings`** on the notification policy.
```